src/main/java/ddingdong/ddingdongBE/domain/vodprocessing/service/FacadeVodProcessingJobServiceImpl.java

```diff
@@ -16,19 +24,42 @@ public class FacadeVodProcessingJobServiceImpl implements FacadeVodProcessingJob
 
     private final VodProcessingJobService vodProcessingJobService;
     private final FileMetaDataService fileMetaDataService;
+    private final VodProcessingNotificationService vodProcessingNotificationService;
+    private final SseConnectionService sseConnectionService;
+    private final FeedService feedService;
 
     @Override
     @Transactional
     public Long create(CreatePendingVodProcessingJobCommand command) {
         FileMetaData fileMetaData = fileMetaDataService.getById(command.fileId());
-        return vodProcessingJobService.save(command.toPendingVodProcessingJob(fileMetaData));
+        VodProcessingNotification pendingNotification =
+                vodProcessingNotificationService.save(VodProcessingNotification.pending());
+        return vodProcessingJobService.save(command.toPendingVodProcessingJob(pendingNotification, fileMetaData));
     }
 
     @Override
     @Transactional
     public void updateVodProcessingJobStatus(UpdateVodProcessingJobStatusCommand command) {
         VodProcessingJob vodProcessingJob = vodProcessingJobService.getByConvertJobId(command.convertJobId());
         vodProcessingJob.updateConvertJobStatus(command.status());
+        checkVodProcessingJobStatus(vodProcessingJob);
+    }
+
+    private void checkVodProcessingJobStatus(VodProcessingJob vodProcessingJob) {
+        ConvertJobStatus convertJobStatus = vodProcessingJob.getConvertJobStatus();
+        if (convertJobStatus != ConvertJobStatus.PENDING) {
+            checkExistingFeedAndNotify(convertJobStatus, vodProcessingJob);
+        }
+    }
+
+    private void checkExistingFeedAndNotify(ConvertJobStatus convertJobStatus, VodProcessingJob vodProcessingJob) {
+        Optional<Feed> optionalFeed = feedService.findById(vodProcessingJob.getFileMetaData().getEntityId());
+        if (optionalFeed.isPresent()) {
+            SseEvent<ConvertJobStatus> sseEvent = SseEvent.of("vod-processing", convertJobStatus, LocalDateTime.now());
+            sseConnectionService.send(vodProcessingJob.getUserId(), sseEvent);
+            VodProcessingNotification vodProcessingNotification = vodProcessingJob.getVodProcessingNotification();
+            vodProcessingNotification.updateVodNotificationStatusToSent(LocalDateTime.now());
```

Comment:
p3)
코드가 중복되는 것 같아서, 하나의 메소드로 생성해도 좋을 것 같아요! 항상 같이 실행되는 작업 같네요
음..아니면
```suggestion
            sseConnectionService.sendVodProcessingJob(vodProcessingJob);
            vodProcessingNotificationService.updateVodNotificationStatusToSent(LocalDateTime.now());
```

Reply:
저도 작성하면서 고민했는데 애매한거 같습니다.. send 메서드는 좀더 범용적인 메서드로써 작동하도록 sse 관련 정보 말고는 사용하지 않으려 했는데 결국 현재 시점에선 VodProcessingJob 관련 notifiy만 수행하니 해당 정보를 받는 send() 메서드를 구현하는게 더 나을거 같네요!

Reply:
범용적인 메소드로 send()는 구현하되, 추가로 sendVodProcessingJob()를 만드는 것이 좋아보인다는 것이었습니다!